code/modules/research/designs/medical_designs.dm

```diff
@@ -563,13 +563,48 @@
 	surgery = /datum/surgery/advanced/viral_bonding
 	research_icon_state = "surgery_chest"
 
-/datum/design/surgery/reconstruction
-	name = "Reconstruction"
-	desc = "A surgical procedure that gradually repairs damage done to a body without the assistance of chemicals. Unlike classic medicine, it is effective on corpses."
-	id = "surgery_reconstruction"
-	surgery = /datum/surgery/advanced/reconstruction
+/datum/design/surgery/healing
+	name = "Tend Wounds"
+	desc = "An upgraded version of the original surgery."
 	research_icon_state = "surgery_chest"
```

Comment:
you forgot to set the ID on this one cobby

Reply:
shaking my head on you right now tbh

Reply:
smh I thought it wouldn't be an issue since it's never used, just for typing.